components/brave_user_agent/common/features.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_BRAVE_USER_AGENT_COMMON_FEATURES_H_
+#define BRAVE_COMPONENTS_BRAVE_USER_AGENT_COMMON_FEATURES_H_
+
+#include "base/component_export.h"
+#include "base/feature_list.h"
+
+namespace brave_user_agent {
+namespace features {
+
+COMPONENT_EXPORT(BRAVE_USER_AGENT_COMMON)
+BASE_DECLARE_FEATURE(kUseBraveUserAgent);
+
+bool IsUseBraveUserAgentEnabled();
```

Comment:
Needs a `COMPONENT_EXPORT(BRAVE_USER_AGENT_COMMON)` on this as well if its ever expected to be used?